lambda/app/src/controllers/user.ts

```diff
@@ -1,10 +1,12 @@
 import { models } from '../../../shared/sequelize/models/models';
 import { Session } from '../../../shared/interfaces';
-import { trim, toLower } from 'lodash';
+import { lowcase } from '@lambda-app/helpers/string';
 
 // TODO: refactor here
```

Comment:
does this still need to be here, or is this the refactor

Reply:
that is `TODO` list that we can work on later to refactor the logic below;
as that is not a priority now but rather it is considered as `tech debt` we can address later.